test/examples/ucp_hello_world.c

```diff
@@ -461,16 +485,16 @@ int main(int argc, char **argv)
     ucp_config_print(config, stdout, NULL, UCS_CONFIG_PRINT_CONFIG);
 
     ucp_config_release(config);
-    CHKERR_JUMP(status != UCS_OK, "ucp_init\n", err);
+    CHKERR_JUMP(status != UCS_OK, "ucp_init", err);
 
     worker_params.field_mask  = UCP_WORKER_PARAM_FIELD_THREAD_MODE;
     worker_params.thread_mode = UCS_THREAD_MODE_SINGLE;
 
     status = ucp_worker_create(ucp_context, &worker_params, &ucp_worker);
-    CHKERR_JUMP(status != UCS_OK, "ucp_worker_create\n", err_cleanup);
+    CHKERR_JUMP(status != UCS_OK, "ucp_worker_create", err_cleanup);
 
     status = ucp_worker_get_address(ucp_worker, &local_addr, &local_addr_len);
-    CHKERR_JUMP(status != UCS_OK, "ucp_worker_get_address\n", err_worker);
+    CHKERR_JUMP(status != UCS_OK, "ucp_worker_get_address", err_worker);
```

Comment:
can you pls separate all "\n" and code style (long line) fixes to another PR? or at least other commit?

Reply:
will prepare new PR with the code-style fixes